app/components/page-components/Chatbot/ChatbotBase.vue.ts

```diff
@@ -27,13 +28,18 @@ import { IListOption } from 'components/shared/inputs'
 export default class ChatbotBase extends Vue {
   @Inject() chatbotApiService: ChatbotApiService;
   @Inject() chatbotCommonService: ChatbotCommonService;
+  @Inject() customizationService: CustomizationService;
 
   mounted() {
     // pre-load them to switch between 2 windows
     this.chatbotApiService.fetchDefaultCommands();
     this.chatbotApiService.fetchLinkProtection();
   }
 
+  get nightTheme() {
```

Comment:
use the same naming 'nightMode' or 'nightTheme'

Reply:
its interchangable throughout the repo. Ill change it to nightMode because its more widely used
@holiber